deploy/cdk/lib/manifest-lambda/manifest-lambda-stack.ts

```diff
@@ -0,0 +1,152 @@
+import apigateway = require('@aws-cdk/aws-apigateway')
+import { Effect, PolicyStatement } from '@aws-cdk/aws-iam'
+import { Function, Runtime } from "@aws-cdk/aws-lambda"
+import { CnameRecord } from "@aws-cdk/aws-route53"
+import { Construct, Duration, Fn, Stack, StackProps, Annotations } from "@aws-cdk/core"
+import path = require('path')
+import { FoundationStack } from '../foundation'
+import { AssetHelpers } from '../asset-helpers'
+import { MaintainMetadataStack } from '../maintain-metadata'
+
+export interface IBaseStackProps extends StackProps {
+
+  /**
+   * The name of the foundation stack upon which this stack is dependent
+   */
+  readonly foundationStack: FoundationStack;
+
+  /**
+   * The domain name to use to reference and create Parameter Store parameters
+   */
+  // readonly domainName: string;
+
+  /**
+   * The sentry data source name (DSN)
+   */
+  readonly sentryDsn: string;
+
+  /**
+   * If True, will attempt to create a Route 53 DNS record for the CloudFront.
+   */
+  readonly createDns: boolean;
+
+  /**
+   * Hostname prefix for the manifest manifest lambda
+   */
+  readonly hostnamePrefix: string;
+
+  /** 
+   * The filesystem root where we can find the source code for all our lambdas.  
+   * e.g.  /user/me/source/marble-manifest-pipeline/
+   * The path for each individual lambda will be appended to this.
+   */
+  readonly lambdaCodeRootPath: string;
+
+  /**
+   * The name of the maintain metadata stack upon which this stack is dependent
+   */
+  readonly maintainMetadataStack: MaintainMetadataStack;
+}
+
+export class ManifestLambdaStack extends Stack {
+
+  private static ssmPolicy(keyPath: string): PolicyStatement {
+    return new PolicyStatement({
+      effect: Effect.ALLOW,
+      actions: ["ssm:GetParametersByPath"],
+      resources: [
+        Fn.sub('arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter' + keyPath + '/*'),
+      ],
+    })
+  }
+
```

Comment:
Is this needed? I don't see it referenced.

Reply:
Thank you.  No, it's not needed.  I just removed it.